cli/http_util.rs

```diff
@@ -16,20 +18,46 @@ use deno_runtime::deno_fetch::reqwest::redirect::Policy;
 use deno_runtime::deno_fetch::reqwest::Client;
 use deno_runtime::deno_fetch::reqwest::StatusCode;
 use log::debug;
+use rustls::ClientConfig;
+use rustls_native_certs::load_native_certs;
 use std::collections::HashMap;
 
 /// Create new instance of async reqwest::Client. This client supports
 /// proxies and doesn't follow redirects.
 pub fn create_http_client(
   user_agent: String,
+  ca_stores: Option<Vec<String>>,
   ca_data: Option<Vec<u8>>,
 ) -> Result<Client, AnyError> {
+  let mut tls_config = ClientConfig::new();
+  // tls_config.set_persistence(CLIENT_SESSION_MEMORY_CACHE.clone()); // todo: load a common session memory cache
+  match ca_stores {
+    Some(stores) => {
+      if stores.contains(&"system".to_string()) {
+        tls_config.root_store =
+          load_native_certs().expect("could not load platform certs");
+      }
+      if stores.contains(&"mozilla".to_string()) {
+        tls_config
+          .root_store
+          .add_server_trust_anchors(&webpki_roots::TLS_SERVER_ROOTS);
+      }
```

Comment:
With this setup system certs would always have priority over the built-in ones. This is not good. Priority should be determined based on the order of the arguments in the env var.

Reply:
Ok I did it this way because the `load_native_certs()` returns a cert store and needed to be set, which implies it would need to be set before you can add more... but now that you mention it because of some other changes to the way its being loaded I can probably now just use `.add_server_trust_anchors(load_native_certs().roots)`, then it would allow them to be loaded in any order.

Reply:
I updated it to load the two stores based on the order they appear in the vector